frontend/talentsearch/resources/js/components/experienceAccordion/ExperienceAccordion.tsx

```diff
@@ -0,0 +1,175 @@
+import React from "react";
+import { Accordion } from "@common/components/accordion/Accordion";
+import BriefCaseIcon from "@heroicons/react/solid/BriefcaseIcon";
+import {
+  AwardExperience,
+  CommunityExperience,
+  EducationExperience,
+  PersonalExperience,
+  WorkExperience,
+} from "../../api/generated";
+
+export interface AccordionProps {
+  anExperience: {
+    experienceType: string;
+    // closed accordion info and repeated fields
+    title?: string;
+    awardedDate?: string;
+    issuedBy?: string;
+    role?: string;
+    organization?: string;
+    areaStudy?: string;
+    institution?: string;
+    startDate?: string;
+    endDate?: string;
+    // opened accordion info
+    // awards
+    awardedTo?: string;
+    awardedScope?: string;
+    // community
+    project?: string;
+    // education
+    type?: string;
+    status?: string;
+    thesisTitle?: string;
+    // personal
+    description?: string;
+    // work
+    division?: string;
+    // linked skills
+    experienceSkills?: [];
+  };
+}
+
+const ExperienceAccordion: React.FunctionComponent<AccordionProps> = ({
+  anExperience,
+}) => {
+  // destructuring required due to linting?
+  const {
+    experienceType,
+    title,
+    awardedDate,
+    issuedBy,
+    role,
+    organization,
+    areaStudy,
+    institution,
+    startDate,
+    endDate,
+    awardedTo,
+    awardedScope,
+    project,
+    type,
+    status,
+    thesisTitle,
+    description,
+    division,
+  } = anExperience;
+
+  // experience type is required with 5 possibilities, build different accordion around which type it is
+  let experienceInstance;
+  if (experienceType === "AwardExperience") {
```

Comment:
The type guards mentioned above might be able to be used instead of this big if/else structure.